libs/utils/prefix-version.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Prefix the the version with a "v".
+ *
+ * Examples
+ *   '1.2.3' => 'v1.2.3'
+ *   'v1.2.3' => 'v1.2.3'
+ *   'dev-master' => 'dev-master'
+ */
+module.exports = (version) => {
```

Comment:
I see this `version ? prefixVersion(version) : 'unknown'` everywhere, so I'm thinking how about make this a `version-formatter`, returns `'unknown'` while empty input, use case could be even simpler:

```javascript
const v = require('../utils/version-formatter.js')

const status = v(version)
```

Reply:
@amio That sounds like a good solution. We can also use is everywhere then :)